### PR TITLE
fix: Workaround GHA Post run failure 

### PR DESCRIPTION
**Pull request by engedaam**
Fixes #N/A 

**Description**
- Add workaround provided by GitHub to mitigate post run failures 
- https://github.com/aws/karpenter/actions/runs/6067597873

**How was this change tested?**
- `/karpenter` snapshot

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates 
- [ ] Yes, issue opened: # 
- [x] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by engedaam — COMMENTED** [review]
/karpenter snapshot

**Review by engedaam — COMMENTED** [review]
/karpenter snapshot

File: .github/workflows/e2e.yaml [hunk 1]
```diff
@@ -92,7 +92,10 @@ jobs:
           echo "Using cluster name \"$CLUSTER_NAME\""
           echo CLUSTER_NAME=$CLUSTER_NAME >> $GITHUB_ENV
       - name: create eks cluster '${{ env.CLUSTER_NAME }}'
-        uses: ./.github/actions/e2e/create-cluster
+        # Revert back to local action if either occurs:
+        # 1) Get a Dockerfile that reduces the number of local steps that we run 
+        # 2) Github fixes the issue that is causing this failure
+        uses: aws/karpenter/.github/actions/e2e/create-cluster@main
```
Comment (jonathan-innis) [inline .github/workflows/e2e.yaml:98]: Can this just reference the local path? I don't think we were having issue in the post steps for this workflow here

**Review by engedaam — COMMENTED** [review]
/karpenter snapshot

**Review by engedaam — COMMENTED** [review]
/karpenter snapshot

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

**Pull request merged**